src/nccl_ofi_rdma.c

```diff
@@ -5392,6 +5527,10 @@ static int connect(nccl_net_ofi_ep_t *base_ep,
 		return -EINVAL;
 	};
 
+	nccl_net_ofi_mutex_lock(&comm_cleanup_list_lock);
+	++num_open_comms;
```

Comment:
just make this atomic?

Reply:
I disagree; we already have roadmap for cleaning up locking and going to a large lock to remove all the duplicate locking in Libfabric.  An atomic doesn't help move towards that.

Reply:
Left as-is then.